h2/src/docsrc/help/help.csv

```diff
@@ -4852,6 +4852,24 @@ This method returns a string.
 CALL STRINGENCODE(STRINGDECODE('Lines 1\nLine 2'))
 "
 
+"Functions (String)","BASE64_ENCODE","
+@h2@ BASE64_ENCODE(bytes)
```

Comment:
You missed the optional second argument, it should be documented too with its possible values here and in another function.